### DISCUSSION TITLE
UI for multiple slots per event?

### DISCUSSION THREAD

**Discussion by stblassitude** [pretalx/pretalx#1229] [Ideas] [answered]
I've just learned that a talk/event can have multiple time slots in the data model, but I can't find a user interface for it. Am I looking in the wrong place?

**Comment by rixx** [comment, marked as answer]
You first have to enable the feature by ticking "Slot Count (can sessions be held multiple times?)". Then you have to set the slot count of the session you wish to repeat on the session detail site.

pretalx will generate multiple slot blocks that you can move around in the schedule editor, then – there is no UI yet to change multiple slots via the session page.

**Reply by stblassitude** [reply]
Got it, thanks!
